profile/src/main/kotlin/com/imashnake/animite/profile/tabs/Anime.kt

```diff
@@ -10,15 +13,21 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.util.fastForEach
 import com.imashnake.animite.api.anilist.sanitize.profile.User
+import com.imashnake.animite.api.anilist.type.MediaType
 import com.imashnake.animite.core.ui.LocalPaddings
 import com.imashnake.animite.core.ui.MediaSmall
 import com.imashnake.animite.core.ui.MediaSmallRow
+import com.imashnake.animite.media.MediaPage
 import com.imashnake.animite.core.R as coreR
 
+@OptIn(ExperimentalSharedTransitionApi::class)
 @Composable
 fun AnimeTab(
     mediaCollection: User.MediaCollection?,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    onNavigateToMediaItem: (MediaPage) -> Unit,
+    sharedTransitionScope: SharedTransitionScope,
+    animatedVisibilityScope: AnimatedVisibilityScope,
```

Comment:
```suggestion
    onNavigateToMediaItem: (MediaPage) -> Unit,
    sharedTransitionScope: SharedTransitionScope,
    animatedVisibilityScope: AnimatedVisibilityScope,
    modifier: Modifier = Modifier,
```
modifier should be [first optional param](https://github.com/slackhq/compose-lints/blob/main/docs/rules.md#modifiers-should-have-default-parameters)